Intermittent integration test failure: test_check_telemetry_sticky_targeting

Seen here:

https://app.circleci.com/pipelines/github/mozilla/experimenter/23603/workflows/4a6c1624-fa76-490f-a184-e46f022b3bdb/jobs/79888


```py
=================================== FAILURES ===================================
____________ test_check_telemetry_sticky_targeting[FIREFOX_DESKTOP] ____________
[gw0] linux -- Python 3.8.10 /code/experimenter/tests/integration/.tox/integration-test-nimbus/bin/python

base_url = 'https://nginx/nimbus/'
selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="30445fbe-daac-4331-88e7-778af996fac0")>
kinto_client = <nimbus.kinto.client.KintoClient object at 0x7f8785713670>
slugify = <function slugify.<locals>._slugify at 0x7f87851d8dc0>
experiment_name = 'test_check_telemetry_sticky_targeting[FIREFOX_DESKTOP]7fc1'
experiment_default_data = {'application': 'DESKTOP', 'changelogMessage': 'test updates', 'featureConfigId': 1, 'hypothesis': 'Test Hypothesis', ...}
check_ping_for_experiment = <function fixture_check_ping_for_experiment.<locals>._check_ping_for_experiment at 0x7f8784902e50>
telemetry_event_check = <function fixture_telemetry_event_check.<locals>._telemetry_event_check at 0x7f8784902ee0>
trigger_experiment_loader = <function trigger_experiment_loader.<locals>._trigger_experiment_loader at 0x7f87849028b0>

    @pytest.mark.desktop_enrollment
    @pytest.mark.xdist_group(name="group1")
    def test_check_telemetry_sticky_targeting(
        base_url,
        selenium,
        kinto_client,
        slugify,
        experiment_name,
        experiment_default_data,
        check_ping_for_experiment,
        telemetry_event_check,
        trigger_experiment_loader,
    ):
        about_config = AboutConfig(selenium)
        pref_name = "sticky.targeting.test.pref"
    
        requests.delete("http://ping-server:5000/pings")
        experiment_slug = str(slugify(experiment_name))
        targeting_config_slug = "no_targeting"
        experiment_default_data["targetingConfigSlug"] = targeting_config_slug
        experiment_default_data["featureConfigId"] = 1
        experiment_default_data["referenceBranch"] = {
            "description": "reference branch",
            "name": "Branch 1",
            "ratio": 100,
            "featureValue": "{}",
        }
        experiment_default_data["treatmentBranches"] = []
        experiment_default_data["isSticky"] = True
        helpers.create_desktop_experiment(
            experiment_slug,
            "desktop",
            targeting_config_slug,
            experiment_default_data,
        )
    
        summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
        summary.launch_and_approve()
    
        kinto_client.approve()
    
        summary = SummaryPage(selenium, urljoin(base_url, experiment_slug)).open()
        summary.wait_for_live_status()
    
        # Ping the server twice as it sleeps sometimes
        requests.get("http://ping-server:5000/pings")
        time.sleep(5)
    
        # Check there was a telemetry event for the enrollment
        control = False
        timeout = time.time() + 60 * 5
        while not control:
            control = telemetry_event_check(experiment_slug, "enroll")
            if time.time() > timeout:
                assert False, "Experiment enrollment was never seen in ping Data"
        # check experiment exists, this means it is enrolled
        assert check_ping_for_experiment(experiment_slug), "Experiment not found in telemetry"
    
        # flip pref
        about_config = about_config.open().wait_for_page_to_load()
        about_config.wait_for_pref_flip(
            pref_name, True, action=trigger_experiment_loader, pref_type=bool
        )
        about_config.flip_pref(pref_name)
    
        assert about_config.get_pref_value(pref_name) == "false"
    
        # check experiment doesn't unenroll after pref flip
        control = False
        timeout = time.time() + 60
        while not control and time.time() < timeout:
            control = telemetry_event_check(experiment_slug, "unenroll")
            if control:
>               assert False, "Experiment unenrolled when it shouldn't have"
E               AssertionError: Experiment unenrolled when it shouldn't have
E               assert False

nimbus/test_desktop_client_integrations.py:312: AssertionError
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3441)

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -222,7 +222,7 @@ jobs:
     working_directory: ~/experimenter
     environment:
       FIREFOX_VERSION: nimbus-firefox-release
-      PYTEST_ARGS: -k FIREFOX_DESKTOP -m desktop_enrollment --dist=loadgroup -n=2
+      PYTEST_ARGS: -k FIREFOX_DESKTOP -m desktop_enrollment
       UPDATE_FIREFOX_VERSION: true
     steps:
       - checkout

--- a/.gitignore
+++ b/.gitignore
@@ -88,6 +88,7 @@ experimenter/experimenter/reporting/reporting-ui/assets/
 .testmondata
 .tmontmp
 experimenter/coverage_html_report/
+experimenter/tests/integration/test-reports/
 
 # Dependencies
 node_modules

